lstchain/data/lstchain_standard_config.json

```diff
@@ -287,13 +287,14 @@
   "write_interleaved_events":{
     "DataWriter": {
       "overwrite":  true,
-      "write_images": false,
-      "write_parameters": false,
-      "write_waveforms": true,
+      "write_dl1_images": false,
+      "write_dl1_parameters": false,
+      "write_r0_waveforms": false,
+      "write_r1_waveforms": true,
       "transform_waveform": true,
       "waveform_dtype": "uint16",
       "waveform_offset": 400,
-      "waveform_scale": 80  
+      "waveform_scale": 60  
```

Comment:
@FrancaCassol Fine with changing the scale here?

Should we change the offset to 300 so that it is equivlent to the 60/5 we use in the EVB?